wp-content/themes/core/pcss/content/panel/_hero.pcss

```diff
@@ -7,6 +7,42 @@
 .site-panel--hero {
 	position: relative;
 
+	.t-content h1 {
+		margin-top: 0;
+	}
+
+	@media (--viewport-medium) {
+		min-height: 75vh;
+	}
+}
+
+/* CASE: Split Right/Left Layout */
+.hero--layout-content-split-left,
+.hero--layout-content-split-right {
+	width: 100%;
+	display: flex;
+	flex-flow: column nowrap;
+
+	@media (--viewport-medium) {
+		flex-flow: row nowrap;
+
+		.c-image__bg {
```

Comment:
It'd be sweet to have this available as an image component modifier class as this is a very common pattern. Heck it'd be sweet to also turn the overlay, AR pattern + particular AR's into an image component modifier classes!

Reply:
Looks like we're already doing this by default here: https://github.com/moderntribe/square-one/blob/5927239f9093d16fc8ff7f5f27c48660e4b5d355/wp-content/themes/core/pcss/components/_image.pcss#L19

I'll add the overlay modifier.

Reply:
Here's the overlay modifier class commit: https://github.com/moderntribe/square-one/pull/360/commits/8a921f4621a6903c1d15878a52c3729cde3035ad